NAMESPACE

```diff
@@ -1,10 +1,12 @@
 # Generated by roxygen2: do not edit by hand
 
 export(grpc_version)
-export(run)
-import(Rcpp)
+export(read_services)
+export(start_server)
```

Comment:
I'm not good at naming things ... but will we have other server fns too?
If not, why not use `grpc_server` and `grpc_client` as the main function names? Or simple `serve` and `request`